gtsfm/evaluation/compare_colmap_outputs.py

```diff
@@ -0,0 +1,93 @@
+"""Script to compare two reconstructions in Colmap's output format.
+
+Authors: Ayush Baid
+"""
+
+import argparse
+import os
+from typing import Dict, List
+from pathlib import Path
+
+from gtsam import Pose3
+
+import gtsfm.runner.gtsfm_runner_base as runner_base
+import gtsfm.utils.geometry_comparisons as geometry_comparisons
+import gtsfm.utils.io as io_utils
+import gtsfm.utils.metrics as metric_utils
+from gtsfm.evaluation.metrics import GtsfmMetricsGroup
+
+
+def load_poses(colmap_dirpath: str) -> Dict[str, Pose3]:
+    wTi_list, image_fnames = io_utils.read_images_txt(fpath=os.path.join(colmap_dirpath, "images.txt"))
+
+    poses: Dict[str, Pose3] = {}
+    for wTi, fname in zip(wTi_list, image_fnames):
+        if wTi is None or fname is None:
+            continue
+
+        poses[fname] = wTi
+
+    return poses
+
+
+def compare_poses(baseline_dirpath: str, current_dirpath: str, output_path: str):
```

Comment:
nit: missing return type

could we also add an `Args` section describing the arguments here?

Reply:
+1

Reply:
Thanks, marking as resolved.